styles/components/avatar.css

```diff
@@ -44,3 +76,33 @@
   width: 125px;
   height: 125px;
 }
+
+/* CSS for Badge Only */
+
+.icon {
+  width: 32px;
+  position: relative;
+  height: 32px;
+}
+.badge {
+  position: absolute;
+  border-radius: 50%;
+  width: 18%;
+  height: 18%;
+}
+.badge--dot {
+  width: 25%;
+  height: 25%;
+}
+.icon > svg {
```

Comment:
you can also give a specific class to SVG instead of targeting it directly.

Reply:
yes but I'm using a third party icon library so I don't have any option to add class name to svgs